.github/workflows/publish-c-apidocs.yml

```diff
@@ -5,11 +5,17 @@ on:
       - main
     paths:
       - include/onnxruntime/core/session
-
   workflow_dispatch:
```

Comment:
should we also add a scheduled trigger for the API doc workflows so the generated artifact gets refreshed and doesn't risk exceeding the retention period? though I think the retention period is 90 days by default so it doesn't need to be very frequent

Reply:
That's a good idea. The default is actually 1 day!

Reply:
Added